In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:134:31: warning: 'Uint32Value' is deprecated [-Wdeprecated-declarations]
      ? optionsObj->Get(opt)->Uint32Value()
                              ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/v8.h:2477:3: note: 'Uint32Value' has been explicitly marked deprecated here
  V8_DEPRECATED("Use maybe version", uint32_t Uint32Value() const);
  ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/v8config.h:327:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated))
                            ^
In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:213:5: error: no member named 'ThrowException' in namespace 'v8'
    _NAN_THROW_ERROR(v8::Exception::Error, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:11: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
      ~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:213:5: error: no member named 'New' in 'v8::String'
    _NAN_THROW_ERROR(v8::Exception::Error, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:42: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
                             ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:218:9: error: no type named 'ThrowException' in namespace 'v8'
    v8::ThrowException(error);
    ~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:222:65: error: no member named 'New' in 'v8::String'
    v8::Local<v8::Value> err = v8::Exception::Error(v8::String::New(msg));
                                                    ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:223:50: error: expected '(' for function-style cast or type construction
    v8::Local<v8::Object> obj = err.As<v8::Object>();
                                       ~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:223:52: error: expected expression
    v8::Local<v8::Object> obj = err.As<v8::Object>();
                                                   ^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:224:65: error: too few arguments to function call, expected 2, have 1
    obj->Set(v8::String::New("code"), v8::Int32::New(errorNumber));
                                      ~~~~~~~~~~~~~~            ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/v8.h:3048:3: note: 'New' declared here
  static Local<Integer> New(Isolate* isolate, int32_t value);
  ^
In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:224:26: error: no member named 'New' in 'v8::String'
    obj->Set(v8::String::New("code"), v8::Int32::New(errorNumber));
             ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:229:5: error: no member named 'ThrowException' in namespace 'v8'
    _NAN_THROW_ERROR(v8::Exception::TypeError, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:11: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
      ~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:229:5: error: no member named 'New' in 'v8::String'
    _NAN_THROW_ERROR(v8::Exception::TypeError, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:42: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
                             ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:233:5: error: no member named 'ThrowException' in namespace 'v8'
    _NAN_THROW_ERROR(v8::Exception::RangeError, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:11: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
      ~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:233:5: error: no member named 'New' in 'v8::String'
    _NAN_THROW_ERROR(v8::Exception::RangeError, errmsg);
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:209:42: note: expanded from macro '_NAN_THROW_ERROR'
      v8::ThrowException(fun(v8::String::New(errmsg)));                        \
                             ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:244:13: error: no member named 'smalloc' in namespace 'node'
      node::smalloc::FreeCallback callback,
      ~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:251:12: error: no matching function for call to 'New'
    return node::Buffer::New(data, size);
           ^~~~~~~~~~~~~~~~~
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:53:40: note: candidate function not viable: no known conversion from 'char *' to 'v8::Isolate *' for 1st argument
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate, size_t length);
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:56:40: note: candidate function not viable: no known conversion from 'char *' to 'v8::Isolate *' for 1st argument
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:68:40: note: candidate function not viable: requires 3 arguments, but 2 were provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:61:40: note: candidate function not viable: requires 5 arguments, but 2 were provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:255:12: error: no matching function for call to 'New'
    return node::Buffer::New(size);
           ^~~~~~~~~~~~~~~~~
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:53:40: note: candidate function not viable: requires 2 arguments, but 1 was provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate, size_t length);
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:56:40: note: candidate function not viable: requires at least 2 arguments, but 1 was provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:68:40: note: candidate function not viable: requires 3 arguments, but 1 was provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/node_buffer.h:61:40: note: candidate function not viable: requires 5 arguments, but 1 was provided
NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                       ^
In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:259:26: error: no member named 'Use' in namespace 'node::Buffer'
    return node::Buffer::Use(data, size);
           ~~~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:450:49: error: too few arguments to function call, single argument 'isolate' was not specified
    v8::Local<v8::Object> obj = v8::Object::New();
                                ~~~~~~~~~~~~~~~ ^
/Users/monsterooo/Library/Caches/node-gyp/10.18.0/include/node/v8.h:3651:3: note: 'New' declared here
  static Local<Object> New(Isolate* isolate);
  ^
In file included from ../src/bufferutil.cc:15:
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:451:14: error: no member named 'NewSymbol' in 'v8::String'
    obj->Set(NanSymbol("callback"), fn);
             ^~~~~~~~~~~~~~~~~~~~~
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:103:38: note: expanded from macro 'NanSymbol'
#define NanSymbol(value) v8::String::NewSymbol(value)
                         ~~~~~~~~~~~~^
/Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/nan/nan.h:457:12: error: no member named 'Dispose' in 'v8::Persistent<v8::Object, v8::NonCopyablePersistentTraits<v8::Object> >'
    handle.Dispose();
    ~~~~~~ ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
1 warning and 20 errors generated.
make: *** [Release/obj.target/bufferutil/src/bufferutil.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/Cellar/node@10/10.18.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:191:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 18.0.0
gyp ERR! command "/usr/local/Cellar/node@10/10.18.0/bin/node" "/usr/local/Cellar/node@10/10.18.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/monsterooo/Desktop/git/ot/node_modules/engine.io-client/node_modules/ws
gyp ERR! node -v v10.18.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
